ci_framework/roles/cifmw_cephadm/tasks/rgw.yml

```diff
@@ -29,6 +29,20 @@
     _hosts: "{{ _hosts|default([]) + [ hostvars[item][ceph_hostname_var] ] }}"
   loop: "{{ groups['edpm'] }}"
 
+- name: Get and save TLS certificate for rgw spec if provided
+  become: true
+  block:
+    - name : Get the certificate content
+      register: slurp_cert
+      ansible.builtin.slurp:
+        src: "{{ cifmw_cephadm_certificate }}"
+
+    - name : Set rgw_frontend_cert for rgw spec
+      set_fact:
+        rgw_frontend_cert: "{{ slurp_cert.get('content', '') | b64decode }}"
```

Comment:
at this point we can go directly with `slurp_cert['content'] | b64decode` - there will be a content, unless previous task fails.